cli/registry/registry.go

```diff
@@ -30,3 +37,47 @@ var Command = &cli.Command{
 		registryListCmd,
 	},
 }
+
+func parseTargetArgs(client woodpecker.Client, c *cli.Context) (global bool, orgID, repoID int64, err error) {
+	if c.Bool("global") {
+		return true, -1, -1, nil
+	}
+
+	repoIDOrFullName := c.String("repository")
+	if repoIDOrFullName == "" {
+		repoIDOrFullName = c.Args().First()
+	}
+
+	orgIDOrName := c.String("organization")
+	if orgIDOrName == "" && repoIDOrFullName == "" {
```

Comment:
What do you think about using sub-commands like `woodpecker-cli org regsitry ...` to have a similar "path" like the UI where you also go to org settings to change an orgs registry. This would also heavily improve navigation through shell completion IMO.

(I know this is done for secrets like this and would need to be refactored as well)

Reply:
this should probably be part of bigger refactoring than

Reply:
Would like to have it done in that way directly to not have a breaking change. Might be best to extract the cli commands to a separate PR.

Reply:
@anbraten refactored CLI commands to subcommands